ubyssey/templates/article/blank.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
     <main class="page">
         <div class="u-container u-container--content">
-          {{ page.get_html|safe }}
+          {{ page.html|safe }}
```

Comment:
My understanding:
Safe means it doesn't need escaping. 

everything in Publishable.html is returned by content_to_html  and is carefully generated. 
I think escaping it would break the generated html?

Removing the safe tag makes no noticeable different however (ie. doesn't break the code embeds and such), so it can be removed.

Made me realize other escaping issues with the content editor in dispatch... which I'll bring up separately.